### PR TITLE
Add airframe (only for JVM projects) to the community build

### PR DESCRIPTION
**Pull request by xerial**
@SethTisue This is an initial attempt to add Airframe (https://github.com/wvlet/airframe) to the community build, as we talked at Scale By The Bay.

Let me know if we need some fixes. 

Thanks.

### CODE REVIEW

File: configs/project-refs.conf [hunk 1]
```diff
@@ -1,5 +1,6 @@
 vars.uris: {
   acyclic-uri:                    "https://github.com/lihaoyi/acyclic.git#cfaa2465105871e41681985c325ce262d1a6460a"
+  airframe-url:                   "https://github.com/wvlet/airframe.git#46b58ef6d0322e4e02d3921b6ad43638f23cf0f9"
```
Comment (xerial) [inline configs/project-refs.conf]: Pointing to the latest master HEAD revision.

Comment (SethTisue) [reply configs/project-refs.conf]: s/url/uri/

I didn't spot the discrepancy, but Jenkins did: https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-integrate-community-build/3782/console

Comment (xerial) [reply configs/project-refs.conf]: ok. Fixed

**Comment by SethTisue** [comment]
```
[airframe] [warn] Originally requested: projectJVM
[airframe] [warn] Now added: finagle, http, config, tablet, fluentd, launcher, codecJVM, stream, jmx, jdbc, airframeJVM, surfaceJVM, jsonBenchmark, jsonJVM, controlJVM, metricsJVM, msgpackJVM, airframeSpecJVM, logJVM, airframeMacrosJVM
```

we'll have to trim some of these subprojects, I assume. are there any you can immediately see ought to be added to `extra.exclude`?

we can also go by trial and error and just see what works and what doesn't, but it's a bit time-consuming because dbuild will bail as soon as any subproject fails, so it's better to cut down the project list in advance if we can

**Comment by SethTisue** [comment]
at https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-integrate-community-build/3784/consoleFull , some subprojects are green already: airframeMacrosJVM, logJVM, metricsJVM, controlJVM, jsonJVM  👍 

```
[airframe] Processing subproject: jsonBenchmark
[airframe] [error] **** Missing dependency: the library io.circe#circe-parser is not provided (in space "scala.main") by any project in this configuration file.
```

so we see that jsonBenchmark failed because it needs circe. we have circe in the community build, but in a different dbuild "space" (section of the config file), but we normally don't include benchmarks in the community build anyway

**Comment by xerial** [comment]
ok. jsonBenchmark doesn’t need to be included (because it’s just for performance testing). jdbc sub project requires PostgreSQL for testing, so we should remove this as well.

**Comment by SethTisue** [comment]
https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-integrate-community-build/3786/consoleFull

```
[airframe] Processing subproject: fluentd
[airframe] [error] **** Missing dependency: the library org.xerial#fluentd-standalone is not provided (in space "scala.main") by any project in this configuration file.
```

**Comment by SethTisue** [comment]
thanks for the PR. pleased to meet you!

**Comment by SethTisue** [comment]
haha oops, I was looking at the wrong log, this wasn't actually green yet.

fixup (excluded finagle subproject): f1859e2e20deeb01e4bb7189315db7013ab91d94

and now it is actually green, at https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-integrate-community-build/3791/consoleFull 👍

**Comment by xerial** [comment]
@SethTisue Thanks! For future improvement, I’ll create an aggregated project for the community build (e.g., communityBuildJVM)

**Pull request merged**